### ISSUE TITLE
From VT: A length of 9-14, a-z, tlds: [com], the sixth new seed of FakeAV?

### ISSUE THREAD

**Issue by suqitian** [360netlab/DGA#23]
- MD5
  [8df8b0115c9f3142591a610ba8716d9c](https://github.com/360netlab/DGA/blob/master/md5/8df8b0115c9f3142591a610ba8716d9c.gz)
- Domains
  See below.
- Looks like a time independent DGA.

**Comment by suqitian** [comment]
All domains, 115 in total.

benypatubeqil.com
bipojizikagec.com
bodylarozityd.com
buderibojine.com
bujycuvoh.com
bunizywytyg.com
byruloqoxybed.com
byxejemaz.com
caxygosob.com
cilywelohy.com
cukumoqurehuj.com
daralytagyc.com
dexiwotenelex.com
dinifuxariqi.com
dopifoqetucol.com
dotecukihilavy.com
dugecafajibox.com
fakiwijow.com
fakukabucom.com
fehosoxukyk.com
firosafemone.com
fudejivuqaran.com
gofegucobeqevi.com
gubebyzosibec.com
hafapuqyse.com
hazovagugaze.com
hejyrabovafy.com
hiqalotajadyfa.com
hireremexyd.com
holytewuzejuk.com
hovucytoc.com
huxeqatikicare.com
jafuwadycylew.com
jicohewihihot.com
jimelyrigupita.com
jiqixylexut.com
josigyhyt.com
jupuqeqej.com
jyxyvikujeq.com
kecupegirekak.com
kovejyvymuzi.com
kyrisapizopu.com
kysumiqehu.com
lagisipiweso.com
lavifopihune.com
lofocigeced.com
lopibobuheqixi.com
meqijuxosomov.com
mijynujidyl.com
mixolyzegito.com
mowecysowo.com
mutidajil.com
myhemilymucy.com
nekehibyfahuf.com
nevehaqyxixocu.com
nicivilokip.com
nipygevydor.com
noqukadifoxe.com
pogavoliqamyb.com
pojizocimovi.com
poqivokimog.com
poquwaluj.com
pozimetycurenu.com
pududigulerewy.com
qehynytezyn.com
qiculeqity.com
qisupikux.com
qybolysusefim.com
qyzywapoz.com
remuvoduq.com
rohyjikyf.com
ryqixafumigeqe.com
sewibonypar.com
sewicyzinuhe.com
socihizizacowo.com
sonewenazo.com
sumywygifi.com
syhyvazemegum.com
takewijejex.com
taxigibyz.com
tewitavubu.com
tikytudububy.com
timypahisoxur.com
tyfifopojax.com
tygemimarowic.com
varolowuqiz.com
vexusysabix.com
vihuhosyde.com
vilifeqimysu.com
vilohezejybyz.com
vivigozymekox.com
vokixehimal.com
vujetejuvax.com
wakuxyvofa.com
warupegacotate.com
wenisekybe.com
wewibolyxov.com
wipujuvajyr.com
wobyfyvovafew.com
wurokalawysusa.com
wyfymabijahiw.com
wygehasunupi.com
wytasosivude.com
wywenybazyxyq.com
xepomumab.com
xijifilunaq.com
xisebozenaj.com
xofokusutecyd.com
xojalyfudux.com
xoxokyliserag.com
zenybijywyrade.com
zinatevyd.com
zutiqasinexoq.com
zuzusutity.com
zyfovubyv.com

**Comment by suqitian** [comment]
Other 4 samples.

2af6cd7d0d766c73c9f4d06b8ffc816f
bab6e96c09c08e5c448d76d290c3c1ca
c1583868b90ea9b03ccdc4626a88a0e4
925ec0eb85b5f17086654ee88560ad70

**Comment by suqitian** [comment]
Not DGA.
